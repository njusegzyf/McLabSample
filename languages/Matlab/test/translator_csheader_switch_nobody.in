switch 1 end

switch f(1) end
switch f (1) end
switch (g(1)) end
switch (g (1)) end

switch f{1} end
switch f {1} end
switch (g{1}) end
switch (g {1}) end

switch u@v end
switch u @v end
switch u @ v end
switch (u@v) end
switch (u @v) end
switch (u@ v) end
switch (u @ v) end

switch u.v end
switch u .v end
switch u. v end
switch u . v end
switch (u.v) end
switch (u .v) end
switch (u. v) end
switch (u . v) end

switch 1.2 end

switch +2 end
switch + 2 end

switch 1+2 end
switch 1 +2 end
switch 1+ 2 end
switch 1 + 2 end
