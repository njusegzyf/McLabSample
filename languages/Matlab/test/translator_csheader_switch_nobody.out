>>>> Destination Language -> Source Language
[1, 1] -> (1, 1)
[1, 2] -> (1, 2)
[1, 3] -> (1, 3)
[1, 4] -> (1, 4)
[1, 5] -> (1, 5)
[1, 6] -> (1, 6)
[1, 7] -> (1, 7)
[1, 8] -> (1, 8)
[1, 9] -> (1, 8)
[1, 10] -> (1, 9)
[1, 11] -> (1, 10)
[1, 12] -> (1, 11)
[1, 13] -> (1, 12)
[1, 14] -> (1, 13)
[2, 1] -> (2, 1)
[3, 1] -> (3, 1)
[3, 2] -> (3, 2)
[3, 3] -> (3, 3)
[3, 4] -> (3, 4)
[3, 5] -> (3, 5)
[3, 6] -> (3, 6)
[3, 7] -> (3, 7)
[3, 8] -> (3, 8)
[3, 9] -> (3, 9)
[3, 10] -> (3, 10)
[3, 11] -> (3, 11)
[3, 12] -> (3, 11)
[3, 13] -> (3, 12)
[3, 14] -> (3, 13)
[3, 15] -> (3, 14)
[3, 16] -> (3, 15)
[3, 17] -> (3, 16)
[4, 1] -> (4, 1)
[4, 2] -> (4, 2)
[4, 3] -> (4, 3)
[4, 4] -> (4, 4)
[4, 5] -> (4, 5)
[4, 6] -> (4, 6)
[4, 7] -> (4, 7)
[4, 8] -> (4, 8)
[4, 9] -> (4, 9)
[4, 10] -> (4, 10)
[4, 11] -> (4, 11)
[4, 12] -> (4, 12)
[4, 13] -> (4, 12)
[4, 14] -> (4, 13)
[4, 15] -> (4, 14)
[4, 16] -> (4, 15)
[4, 17] -> (4, 16)
[4, 18] -> (4, 17)
[5, 1] -> (5, 1)
[5, 2] -> (5, 2)
[5, 3] -> (5, 3)
[5, 4] -> (5, 4)
[5, 5] -> (5, 5)
[5, 6] -> (5, 6)
[5, 7] -> (5, 7)
[5, 8] -> (5, 8)
[5, 9] -> (5, 9)
[5, 10] -> (5, 10)
[5, 11] -> (5, 11)
[5, 12] -> (5, 12)
[5, 13] -> (5, 13)
[5, 14] -> (5, 13)
[5, 15] -> (5, 14)
[5, 16] -> (5, 15)
[5, 17] -> (5, 16)
[5, 18] -> (5, 17)
[5, 19] -> (5, 18)
[6, 1] -> (6, 1)
[6, 2] -> (6, 2)
[6, 3] -> (6, 3)
[6, 4] -> (6, 4)
[6, 5] -> (6, 5)
[6, 6] -> (6, 6)
[6, 7] -> (6, 7)
[6, 8] -> (6, 8)
[6, 9] -> (6, 9)
[6, 10] -> (6, 10)
[6, 11] -> (6, 11)
[6, 12] -> (6, 12)
[6, 13] -> (6, 13)
[6, 14] -> (6, 14)
[6, 15] -> (6, 14)
[6, 16] -> (6, 15)
[6, 17] -> (6, 16)
[6, 18] -> (6, 17)
[6, 19] -> (6, 18)
[6, 20] -> (6, 19)
[7, 1] -> (7, 1)
[8, 1] -> (8, 1)
[8, 2] -> (8, 2)
[8, 3] -> (8, 3)
[8, 4] -> (8, 4)
[8, 5] -> (8, 5)
[8, 6] -> (8, 6)
[8, 7] -> (8, 7)
[8, 8] -> (8, 8)
[8, 9] -> (8, 9)
[8, 10] -> (8, 10)
[8, 11] -> (8, 11)
[8, 12] -> (8, 11)
[8, 13] -> (8, 12)
[8, 14] -> (8, 13)
[8, 15] -> (8, 14)
[8, 16] -> (8, 15)
[8, 17] -> (8, 16)
[9, 1] -> (9, 1)
[9, 2] -> (9, 2)
[9, 3] -> (9, 3)
[9, 4] -> (9, 4)
[9, 5] -> (9, 5)
[9, 6] -> (9, 6)
[9, 7] -> (9, 7)
[9, 8] -> (9, 8)
[9, 9] -> (9, 9)
[9, 10] -> (9, 10)
[9, 11] -> (9, 11)
[9, 12] -> (9, 12)
[9, 13] -> (9, 12)
[9, 14] -> (9, 13)
[9, 15] -> (9, 14)
[9, 16] -> (9, 15)
[9, 17] -> (9, 16)
[9, 18] -> (9, 17)
[10, 1] -> (10, 1)
[10, 2] -> (10, 2)
[10, 3] -> (10, 3)
[10, 4] -> (10, 4)
[10, 5] -> (10, 5)
[10, 6] -> (10, 6)
[10, 7] -> (10, 7)
[10, 8] -> (10, 8)
[10, 9] -> (10, 9)
[10, 10] -> (10, 10)
[10, 11] -> (10, 11)
[10, 12] -> (10, 12)
[10, 13] -> (10, 13)
[10, 14] -> (10, 13)
[10, 15] -> (10, 14)
[10, 16] -> (10, 15)
[10, 17] -> (10, 16)
[10, 18] -> (10, 17)
[10, 19] -> (10, 18)
[11, 1] -> (11, 1)
[11, 2] -> (11, 2)
[11, 3] -> (11, 3)
[11, 4] -> (11, 4)
[11, 5] -> (11, 5)
[11, 6] -> (11, 6)
[11, 7] -> (11, 7)
[11, 8] -> (11, 8)
[11, 9] -> (11, 9)
[11, 10] -> (11, 10)
[11, 11] -> (11, 11)
[11, 12] -> (11, 12)
[11, 13] -> (11, 13)
[11, 14] -> (11, 14)
[11, 15] -> (11, 14)
[11, 16] -> (11, 15)
[11, 17] -> (11, 16)
[11, 18] -> (11, 17)
[11, 19] -> (11, 18)
[11, 20] -> (11, 19)
[12, 1] -> (12, 1)
[13, 1] -> (13, 1)
[13, 2] -> (13, 2)
[13, 3] -> (13, 3)
[13, 4] -> (13, 4)
[13, 5] -> (13, 5)
[13, 6] -> (13, 6)
[13, 7] -> (13, 7)
[13, 8] -> (13, 8)
[13, 9] -> (13, 9)
[13, 10] -> (13, 10)
[13, 11] -> (13, 10)
[13, 12] -> (13, 11)
[13, 13] -> (13, 12)
[13, 14] -> (13, 13)
[13, 15] -> (13, 14)
[13, 16] -> (13, 15)
[14, 1] -> (14, 1)
[14, 2] -> (14, 2)
[14, 3] -> (14, 3)
[14, 4] -> (14, 4)
[14, 5] -> (14, 5)
[14, 6] -> (14, 6)
[14, 7] -> (14, 7)
[14, 8] -> (14, 8)
[14, 9] -> (14, 9)
[14, 10] -> (14, 10)
[14, 11] -> (14, 11)
[14, 12] -> (14, 11)
[14, 13] -> (14, 12)
[14, 14] -> (14, 13)
[14, 15] -> (14, 14)
[14, 16] -> (14, 15)
[14, 17] -> (14, 16)
[15, 1] -> (15, 1)
[15, 2] -> (15, 2)
[15, 3] -> (15, 3)
[15, 4] -> (15, 4)
[15, 5] -> (15, 5)
[15, 6] -> (15, 6)
[15, 7] -> (15, 7)
[15, 8] -> (15, 8)
[15, 9] -> (15, 9)
[15, 10] -> (15, 10)
[15, 11] -> (15, 11)
[15, 12] -> (15, 12)
[15, 13] -> (15, 12)
[15, 14] -> (15, 13)
[15, 15] -> (15, 14)
[15, 16] -> (15, 15)
[15, 17] -> (15, 16)
[15, 18] -> (15, 17)
[16, 1] -> (16, 1)
[16, 2] -> (16, 2)
[16, 3] -> (16, 3)
[16, 4] -> (16, 4)
[16, 5] -> (16, 5)
[16, 6] -> (16, 6)
[16, 7] -> (16, 7)
[16, 8] -> (16, 8)
[16, 9] -> (16, 9)
[16, 10] -> (16, 10)
[16, 11] -> (16, 11)
[16, 12] -> (16, 12)
[16, 13] -> (16, 12)
[16, 14] -> (16, 13)
[16, 15] -> (16, 14)
[16, 16] -> (16, 15)
[16, 17] -> (16, 16)
[16, 18] -> (16, 17)
[17, 1] -> (17, 1)
[17, 2] -> (17, 2)
[17, 3] -> (17, 3)
[17, 4] -> (17, 4)
[17, 5] -> (17, 5)
[17, 6] -> (17, 6)
[17, 7] -> (17, 7)
[17, 8] -> (17, 8)
[17, 9] -> (17, 9)
[17, 10] -> (17, 10)
[17, 11] -> (17, 11)
[17, 12] -> (17, 12)
[17, 13] -> (17, 13)
[17, 14] -> (17, 13)
[17, 15] -> (17, 14)
[17, 16] -> (17, 15)
[17, 17] -> (17, 16)
[17, 18] -> (17, 17)
[17, 19] -> (17, 18)
[18, 1] -> (18, 1)
[18, 2] -> (18, 2)
[18, 3] -> (18, 3)
[18, 4] -> (18, 4)
[18, 5] -> (18, 5)
[18, 6] -> (18, 6)
[18, 7] -> (18, 7)
[18, 8] -> (18, 8)
[18, 9] -> (18, 9)
[18, 10] -> (18, 10)
[18, 11] -> (18, 11)
[18, 12] -> (18, 12)
[18, 13] -> (18, 13)
[18, 14] -> (18, 13)
[18, 15] -> (18, 14)
[18, 16] -> (18, 15)
[18, 17] -> (18, 16)
[18, 18] -> (18, 17)
[18, 19] -> (18, 18)
[19, 1] -> (19, 1)
[19, 2] -> (19, 2)
[19, 3] -> (19, 3)
[19, 4] -> (19, 4)
[19, 5] -> (19, 5)
[19, 6] -> (19, 6)
[19, 7] -> (19, 7)
[19, 8] -> (19, 8)
[19, 9] -> (19, 9)
[19, 10] -> (19, 10)
[19, 11] -> (19, 11)
[19, 12] -> (19, 12)
[19, 13] -> (19, 13)
[19, 14] -> (19, 14)
[19, 15] -> (19, 14)
[19, 16] -> (19, 15)
[19, 17] -> (19, 16)
[19, 18] -> (19, 17)
[19, 19] -> (19, 18)
[19, 20] -> (19, 19)
[20, 1] -> (20, 1)
[21, 1] -> (21, 1)
[21, 2] -> (21, 2)
[21, 3] -> (21, 3)
[21, 4] -> (21, 4)
[21, 5] -> (21, 5)
[21, 6] -> (21, 6)
[21, 7] -> (21, 7)
[21, 8] -> (21, 8)
[21, 9] -> (21, 9)
[21, 10] -> (21, 10)
[21, 11] -> (21, 10)
[21, 12] -> (21, 11)
[21, 13] -> (21, 12)
[21, 14] -> (21, 13)
[21, 15] -> (21, 14)
[21, 16] -> (21, 15)
[22, 1] -> (22, 1)
[22, 2] -> (22, 2)
[22, 3] -> (22, 3)
[22, 4] -> (22, 4)
[22, 5] -> (22, 5)
[22, 6] -> (22, 6)
[22, 7] -> (22, 7)
[22, 8] -> (22, 8)
[22, 9] -> (22, 9)
[22, 10] -> (22, 10)
[22, 11] -> (22, 11)
[22, 12] -> (22, 11)
[22, 13] -> (22, 12)
[22, 14] -> (22, 13)
[22, 15] -> (22, 14)
[22, 16] -> (22, 15)
[22, 17] -> (22, 16)
[23, 1] -> (23, 1)
[23, 2] -> (23, 2)
[23, 3] -> (23, 3)
[23, 4] -> (23, 4)
[23, 5] -> (23, 5)
[23, 6] -> (23, 6)
[23, 7] -> (23, 7)
[23, 8] -> (23, 8)
[23, 9] -> (23, 9)
[23, 10] -> (23, 10)
[23, 11] -> (23, 11)
[23, 12] -> (23, 11)
[23, 13] -> (23, 12)
[23, 14] -> (23, 13)
[23, 15] -> (23, 14)
[23, 16] -> (23, 15)
[23, 17] -> (23, 16)
[24, 1] -> (24, 1)
[24, 2] -> (24, 2)
[24, 3] -> (24, 3)
[24, 4] -> (24, 4)
[24, 5] -> (24, 5)
[24, 6] -> (24, 6)
[24, 7] -> (24, 7)
[24, 8] -> (24, 8)
[24, 9] -> (24, 9)
[24, 10] -> (24, 10)
[24, 11] -> (24, 11)
[24, 12] -> (24, 12)
[24, 13] -> (24, 12)
[24, 14] -> (24, 13)
[24, 15] -> (24, 14)
[24, 16] -> (24, 15)
[24, 17] -> (24, 16)
[24, 18] -> (24, 17)
[25, 1] -> (25, 1)
[25, 2] -> (25, 2)
[25, 3] -> (25, 3)
[25, 4] -> (25, 4)
[25, 5] -> (25, 5)
[25, 6] -> (25, 6)
[25, 7] -> (25, 7)
[25, 8] -> (25, 8)
[25, 9] -> (25, 9)
[25, 10] -> (25, 10)
[25, 11] -> (25, 11)
[25, 12] -> (25, 12)
[25, 13] -> (25, 12)
[25, 14] -> (25, 13)
[25, 15] -> (25, 14)
[25, 16] -> (25, 15)
[25, 17] -> (25, 16)
[25, 18] -> (25, 17)
[26, 1] -> (26, 1)
[26, 2] -> (26, 2)
[26, 3] -> (26, 3)
[26, 4] -> (26, 4)
[26, 5] -> (26, 5)
[26, 6] -> (26, 6)
[26, 7] -> (26, 7)
[26, 8] -> (26, 8)
[26, 9] -> (26, 9)
[26, 10] -> (26, 10)
[26, 11] -> (26, 11)
[26, 12] -> (26, 12)
[26, 13] -> (26, 13)
[26, 14] -> (26, 13)
[26, 15] -> (26, 14)
[26, 16] -> (26, 15)
[26, 17] -> (26, 16)
[26, 18] -> (26, 17)
[26, 19] -> (26, 18)
[27, 1] -> (27, 1)
[27, 2] -> (27, 2)
[27, 3] -> (27, 3)
[27, 4] -> (27, 4)
[27, 5] -> (27, 5)
[27, 6] -> (27, 6)
[27, 7] -> (27, 7)
[27, 8] -> (27, 8)
[27, 9] -> (27, 9)
[27, 10] -> (27, 10)
[27, 11] -> (27, 11)
[27, 12] -> (27, 12)
[27, 13] -> (27, 13)
[27, 14] -> (27, 13)
[27, 15] -> (27, 14)
[27, 16] -> (27, 15)
[27, 17] -> (27, 16)
[27, 18] -> (27, 17)
[27, 19] -> (27, 18)
[28, 1] -> (28, 1)
[28, 2] -> (28, 2)
[28, 3] -> (28, 3)
[28, 4] -> (28, 4)
[28, 5] -> (28, 5)
[28, 6] -> (28, 6)
[28, 7] -> (28, 7)
[28, 8] -> (28, 8)
[28, 9] -> (28, 9)
[28, 10] -> (28, 10)
[28, 11] -> (28, 11)
[28, 12] -> (28, 12)
[28, 13] -> (28, 13)
[28, 14] -> (28, 14)
[28, 15] -> (28, 14)
[28, 16] -> (28, 15)
[28, 17] -> (28, 16)
[28, 18] -> (28, 17)
[28, 19] -> (28, 18)
[28, 20] -> (28, 19)
[29, 1] -> (29, 1)
[30, 1] -> (30, 1)
[30, 2] -> (30, 2)
[30, 3] -> (30, 3)
[30, 4] -> (30, 4)
[30, 5] -> (30, 5)
[30, 6] -> (30, 6)
[30, 7] -> (30, 7)
[30, 8] -> (30, 8)
[30, 9] -> (30, 9)
[30, 10] -> (30, 10)
[30, 11] -> (30, 10)
[30, 12] -> (30, 11)
[30, 13] -> (30, 12)
[30, 14] -> (30, 13)
[30, 15] -> (30, 14)
[30, 16] -> (30, 15)
[31, 1] -> (31, 1)
[32, 1] -> (32, 1)
[32, 2] -> (32, 2)
[32, 3] -> (32, 3)
[32, 4] -> (32, 4)
[32, 5] -> (32, 5)
[32, 6] -> (32, 6)
[32, 7] -> (32, 7)
[32, 8] -> (32, 8)
[32, 9] -> (32, 9)
[32, 10] -> (32, 9)
[32, 11] -> (32, 10)
[32, 12] -> (32, 11)
[32, 13] -> (32, 12)
[32, 14] -> (32, 13)
[32, 15] -> (32, 14)
[33, 1] -> (33, 1)
[33, 2] -> (33, 2)
[33, 3] -> (33, 3)
[33, 4] -> (33, 4)
[33, 5] -> (33, 5)
[33, 6] -> (33, 6)
[33, 7] -> (33, 7)
[33, 8] -> (33, 8)
[33, 9] -> (33, 9)
[33, 10] -> (33, 10)
[33, 11] -> (33, 10)
[33, 12] -> (33, 11)
[33, 13] -> (33, 12)
[33, 14] -> (33, 13)
[33, 15] -> (33, 14)
[33, 16] -> (33, 15)
[34, 1] -> (34, 1)
[35, 1] -> (35, 1)
[35, 2] -> (35, 2)
[35, 3] -> (35, 3)
[35, 4] -> (35, 4)
[35, 5] -> (35, 5)
[35, 6] -> (35, 6)
[35, 7] -> (35, 7)
[35, 8] -> (35, 8)
[35, 9] -> (35, 9)
[35, 10] -> (35, 10)
[35, 11] -> (35, 10)
[35, 12] -> (35, 11)
[35, 13] -> (35, 12)
[35, 14] -> (35, 13)
[35, 15] -> (35, 14)
[35, 16] -> (35, 15)
[36, 1] -> (36, 1)
[36, 2] -> (36, 2)
[36, 3] -> (36, 3)
[36, 4] -> (36, 4)
[36, 5] -> (36, 5)
[36, 6] -> (36, 6)
[36, 7] -> (36, 7)
[36, 8] -> (36, 8)
[36, 9] -> (36, 9)
[36, 10] -> (36, 10)
[36, 11] -> (36, 11)
[36, 12] -> (36, 11)
[36, 13] -> (36, 12)
[36, 14] -> (36, 13)
[36, 15] -> (36, 14)
[36, 16] -> (36, 15)
[36, 17] -> (36, 16)
[37, 1] -> (37, 1)
[37, 2] -> (37, 2)
[37, 3] -> (37, 3)
[37, 4] -> (37, 4)
[37, 5] -> (37, 5)
[37, 6] -> (37, 6)
[37, 7] -> (37, 7)
[37, 8] -> (37, 8)
[37, 9] -> (37, 9)
[37, 10] -> (37, 10)
[37, 11] -> (37, 11)
[37, 12] -> (37, 11)
[37, 13] -> (37, 12)
[37, 14] -> (37, 13)
[37, 15] -> (37, 14)
[37, 16] -> (37, 15)
[37, 17] -> (37, 16)
[38, 1] -> (38, 1)
[38, 2] -> (38, 2)
[38, 3] -> (38, 3)
[38, 4] -> (38, 4)
[38, 5] -> (38, 5)
[38, 6] -> (38, 6)
[38, 7] -> (38, 7)
[38, 8] -> (38, 8)
[38, 9] -> (38, 9)
[38, 10] -> (38, 10)
[38, 11] -> (38, 11)
[38, 12] -> (38, 12)
[38, 13] -> (38, 12)
[38, 14] -> (38, 13)
[38, 15] -> (38, 14)
[38, 16] -> (38, 15)
[38, 17] -> (38, 16)
[38, 18] -> (38, 17)
>>>> Translated Text
switch 1, end

switch f(1), end
switch f (1), end
switch (g(1)), end
switch (g (1)), end

switch f{1}, end
switch f {1}, end
switch (g{1}), end
switch (g {1}), end

switch u@v, end
switch u @v, end
switch u @ v, end
switch (u@v), end
switch (u @v), end
switch (u@ v), end
switch (u @ v), end

switch u.v, end
switch u .v, end
switch u. v, end
switch u . v, end
switch (u.v), end
switch (u .v), end
switch (u. v), end
switch (u . v), end

switch 1.2, end

switch +2, end
switch + 2, end

switch 1+2, end
switch 1 +2, end
switch 1+ 2, end
switch 1 + 2, end
